clay.cabal

```diff
@@ -88,7 +89,7 @@ Test-Suite Test-Clay
   Build-Tools: hspec-discover
   main-is: Spec.hs
   Build-Depends:
-    base                 >= 4.11  && < 4.17,
+    base                 >= 4.11  && < 4.19,
```

Comment:
Technically we only need `< 4.18` since GHC 9.4 corresponds to `4.17`. (And the same further up.) See https://www.snoyman.com/base/

Reply:
Oops, yes, fixed now.